codecov.yml

```diff
@@ -30,10 +30,10 @@ coverage:
           - mmapi.pm
           - osutils.pm
           - signalblocker.pm
-          - t
           - xt
-        ignore:
-          - t/data/tests
+          - t/*.t
```

Comment:
I don't understand this line change. We *want* to collect the coverage for the tests themselves and ensure 100% so that we know that every test line is actually executed. The other line changes are fine. Does it still work if you revert this line?

Reply:
I don't understand. My change will make sure that all `t/*.t` are covered 100%

Reply:
I can make a PR with uncovered lines to prove it actually works: #2293

Reply:
It worked: `codecov/project/fully_covered Failing after 1s — 99.99% (target 100.00%)`

Reply:
Oh, you actually removed the line "ignore" , I didn't see that 🤣

Reply:
Maybe I should have kept the correct sort order, I think then the diff would have been clearer :)
Did that now, also to trigger OBS checks